charts/spire/charts/spiffe-oidc-discovery-provider/templates/tests/test-connection.yaml

```diff
@@ -7,17 +7,25 @@ metadata:
   annotations:
     "helm.sh/hook": test
 spec:
+  securityContext:
+    {{- toYaml .Values.podSecurityContext | nindent 4 }}
   containers:
     - name: wget-service-name
       image: busybox
       command: ['wget']
-      args: ['{{ include "spiffe-oidc-discovery-provider.fullname" . }}:{{ .Values.service.port }}/.well-known/openid-configuration']
+      args: ['-O', '/dev/null', '{{ include "spiffe-oidc-discovery-provider.fullname" . }}:{{ .Values.service.port }}/.well-known/openid-configuration']
```

Comment:
Why hide the output? That makes it hard to debug the tests isn't it?

Reply:
Its not hiding the output, its writing the file it normally would to the current working directory to dev/null instead. That way the root filesystem can be read only.